src/queries/UserQuery.ts

```diff
@@ -1 +1,16 @@
-export {};
+import { User } from '../models/UserModel'
+
+export const getUserQuery = (username: string) =>
```

Comment:
just call it `getUser` the file name tells what it is.

it should be generic and take options obj.

error should be handled by the caller so no need to wrap this in a try catch, and you should be using asyc/await anyway.

```js
const getUser = ({ filter, attributes }) => User.findOne({ where: filter, attributes, raw: true })
```